src/collection/components/ShareCollectionModal.tsx

```diff
@@ -60,8 +60,17 @@ const ShareCollectionModal: FunctionComponent<ShareCollectionModalProps> = ({
 
 		// Check if fragment has custom_title and custom_description if necessary.
```

Comment:
Ik zou de comments updaten naar:
// Video fragments need to have a title AND a description if use_custom_fields is enabled
// Text fragments need to have either a title OR a description